directjngine/src/main/java/com/softwarementors/extjs/djn/servlet/DirectJNgineServlet.java

```diff
@@ -561,6 +563,15 @@ else if( RequestRouter.isSourceRequest(pathInfo)) {
     }
   }
 
+  public static final boolean isMultipartContent(HttpServletRequest request) {
+    if (!("POST".equalsIgnoreCase(request.getMethod()) || "PUT".equalsIgnoreCase(request.getMethod()))) {
+      return false;
+    }
+    else {
+      return FileUploadBase.isMultipartContent(new ServletRequestContext(request));
```

Comment:
why use ```FileUploadBase.isMultipartContent(new ServletRequestContext(request))``` rather than ```ServletFileUpload.isMultipartContent(request)``` that was there previously ?

Not saying it's wrong, just curious

Reply:
ServletFileUpload.isMultipartContent is only checking for POST requests and I didn't want to fork another library :P